packages/calypso-e2e/src/lib/flows/start-site-flow.ts

```diff
@@ -7,8 +7,8 @@ const selectors = {
 	backLink: 'button:text("Back")',
 
 	// Inputs
-	blogNameInput: 'input#siteTitle:not(:disabled)',
-	taglineInput: 'input#tagline:not(:disabled)',
+	blogNameInput: ( value: string ) => `input[name="siteTitle"][value="${ value }"]`,
```

Comment:
@dpasque This implements the suggestion you made in our chat this morning (June 8, 2022).

Looking at the step duration that Jest reports, it doesn't seem to change much compared to the current (flawed) approach, but it hasn't failed yet either. Here's to hoping 🤞